tools/accuracy_checker/accuracy_checker/representation/segmentation_representation.py

```diff
@@ -50,7 +51,26 @@ class GTMaskLoader(Enum):
 
 
 class SegmentationRepresentation(BaseRepresentation):
-    pass
+    def to_polygon(self, segmentation_colors=None):
+        polygons = {}
+        mask = self._encode_mask(self.mask, segmentation_colors) if segmentation_colors else self.mask
+        if len(mask.shape) == 3:
+            mask = np.argmax(mask, axis=0)
+        indexes = np.unique(mask)
+        for i in indexes:
+            binary_mask = np.uint8(mask == i)
+
+            contours, _ = cv.findContours(binary_mask, cv.RETR_TREE, cv.CHAIN_APPROX_SIMPLE)
+
+            for contour in contours:
+                if contour.size >= 6:
+                    contour = np.squeeze(contour, axis=1)
+                    if i not in polygons:
+                        polygons[i] = [contour]
+                    else:
+                        polygons[i].append(contour)
```

Comment:
it looks like good point for starting usage defaultdict:
https://docs.python.org/2/library/collections.html#collections.defaultdict